docs/5.0/features/enhanced-session-recording.md

```diff
@@ -89,15 +89,52 @@ Linux ip-172-31-43-104.ec2.internal 4.19.72-25.58.amzn2.x86_64 x86_64 x86_64 x86
 
 ## 2. Install BCC Tools
 
-Run the following script to download the prerequisites to build BCC tools, building LLVM and Clang targeting BPF byte code, and then building and installing BCC tools.
+We recommend installing BCC tools using your distribution's package manager wherever possible.
 
-!!! note
+=== "Ubuntu/Debian 18.04+"
+
+    ```sh
+    apt -y install bpfcc-tools
+    ```
+
+=== "CentOS/RHEL 8+"
+
+    ```sh
+    yum -y install bcc-tools
+    ```
 
-    We plan to soon support installing bcc-tools from packages instead of compiling them yourself to make taking advantage of enhanced session recording easier.
```

Comment:
I still had problems with this due to https://github.com/gravitational/teleport/issues/4171 . It doesn't look like we've updated this since? cc @russjones